ios/FluentUI/List/ListItemModifiers.swift

```diff
@@ -63,8 +65,25 @@ public extension ListItem {
     /// - Parameter size: The size the leading content should be.
     /// - Returns: The modified `ListItem` with the property set.
     func leadingContentSize(_ size: ListItemLeadingContentSize) -> ListItem {
-        var listItem = self
-        listItem.tokenSet = ListItemTokenSet(customViewSize: { size })
+        let listItem = self
+        let customViewDimensions: CGFloat
+        let customViewTrailingMargin: CGFloat
+
+        switch size {
+        case .zero:
+            customViewDimensions = 0.0
+            customViewTrailingMargin = GlobalTokens.spacing(.sizeNone)
+        case .small:
+            customViewDimensions = GlobalTokens.icon(.size240)
+            customViewTrailingMargin = GlobalTokens.spacing(.size160)
+        case .medium, .default:
+            customViewDimensions = GlobalTokens.icon(.size400)
+            customViewTrailingMargin = GlobalTokens.spacing(.size120)
+        }
+
+        listItem.tokenSet[.customViewDimensions] = .float { customViewDimensions }
```

Comment:
this may cause a bug where if a new token is added to the tvc or list item token set that is dependent on the `customViewSize`, it's not updated here but I wasn't sure how else to ensure that other overridden tokens don't get mangled here.

Reply:
Could you describe a scenario where this might happen? Trying to understand it better. If this change is merged as-is with this potential bug, I think a comment should be added to call out the behavior.

Reply:
so this would only happen if we added a new token to the `TableViewCellTokenSet` that depended on `customViewSize`. essentially what would happen is that token would never update even though `customViewSize` changed. the current tokens that depend on `customViewSize` work because I'm manually updating them here.

Reply:
however, I talked to Mike and we're going to do the "proper" fix which is ensuring that whenever `customViewSize` updates, all the tokens that rely on it update. so fix coming soon